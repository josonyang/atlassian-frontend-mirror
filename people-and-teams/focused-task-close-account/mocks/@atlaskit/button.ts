const Button = 'Button';
const ButtonGroup = 'ButtonGroup';

export default Button;

export { ButtonGroup };
