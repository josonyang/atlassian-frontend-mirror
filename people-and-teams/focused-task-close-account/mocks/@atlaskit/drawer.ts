const Drawer = 'Drawer';

export default Drawer;
