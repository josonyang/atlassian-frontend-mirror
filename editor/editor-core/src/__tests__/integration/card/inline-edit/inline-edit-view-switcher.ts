import {
  editable,
  getDocFromElement,
} from '@atlaskit/editor-test-helpers/integration/helpers';
import {
  goToEditorTestingWDExample,
  mountEditor,
} from '@atlaskit/editor-test-helpers/testing-example-page';
import { waitForInlineCardSelection } from '@atlaskit/media-integration-test-helpers';
import { BrowserTestCase } from '@atlaskit/webdriver-runner/runner';
import * as inlineCardAdf from '../_fixtures_/inline-card.adf.json';

type ClientType = Parameters<typeof goToEditorTestingWDExample>[0];

BrowserTestCase(
  'card: should be able to switch views with view switcher with icons toolbar',
  {},
  async (client: ClientType, testName: string) => {
    const page = await goToEditorTestingWDExample(client);

    await mountEditor(page, {
      appearance: 'full-page',
      allowTextAlignment: true,
      defaultValue: JSON.stringify(inlineCardAdf),
      smartLinks: {
        allowBlockCards: true,
      },
    });

    await waitForInlineCardSelection(page);
    await page.click('[data-testid="block-appearance"]');

    expect(
      await page.$eval(editable, getDocFromElement),
    ).toMatchCustomDocSnapshot(testName);
  },
);
