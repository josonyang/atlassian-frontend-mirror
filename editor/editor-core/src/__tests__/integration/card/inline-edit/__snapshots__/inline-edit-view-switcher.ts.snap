// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card: should be able to switch views with view switcher with icons toolbar 1`] = `
Object {
  "content": Array [
    Object {
      "attrs": Object {
        "data": null,
        "url": "https://inlineCardTestUrl",
      },
      "type": "blockCard",
    },
    Object {
      "content": Array [
        Object {
          "text": " ",
          "type": "text",
        },
      ],
      "type": "paragraph",
    },
  ],
  "type": "doc",
}
`;
